packages/git-encrypted/src/encryptedMeta.ts

```diff
@@ -0,0 +1,96 @@
+import zod from 'zod';
+import { superpathjoin as join } from 'superpathjoin';
+
+import { FS, GitBaseParamsEncrypted } from './types';
+import { generateKey } from './crypto';
+import { encryptedRepoCommit } from './git';
+
+const META_FILENAME = 'encrypted.json';
```

Comment:
nit: I think ENCRYPTED_META_FILENAME would be clearer.

Also, does this belong in `constants`? Maybe, not super clear to me...

Reply:
I agree, ENCRYPTED_META_FILENAME is clearer. 

My reason to put the constant in this file is that it's only used in this file, and to only put it in `constants` when it is needed somewhere else as well. Not 100% sure if this is the right rationale.